yarn.lock

```diff
@@ -0,0 +1,13514 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
With https://github.com/jina-ai/dashboard/pull/32, this project started using `npm` over `yarn`.

So, having `yarn.lock` file could be confusing for contributors. Please remove it.